UI/src/components/widgets/product/view.js

```diff
@@ -88,100 +88,66 @@
-            },
-            {
-                metricName: "unitTests",
-                displayName: "Unit Test Success",
-                scoreRanges: [{ rangeMin: 0, rangeMax: 99, score: 0 },
-                    { rangeMin: 100, rangeMax: 100, score: 20 }],
-                displaySymbol: "%",
-                description: "This metric is based on whether or not the unit tests are passing."
-            },
-            {
-                metricName: "buildSuccess",
-                displayName: "Build Success",
-                scoreRanges: [{ rangeMin: 0, rangeMax: 10, score: 0 },
-                    { rangeMin: 11, rangeMax: 60, score: 4 },
-                    { rangeMin: 61, rangeMax: 70, score: 8 },
-                    { rangeMin: 71, rangeMax: 80, score: 12 },
-                    { rangeMin: 81, rangeMax: 90, score: 16 },
-                    { rangeMin: 91, rangeMax: 95, score: 18 },
-                    { rangeMin: 96, rangeMax: 100, score: 20 }],
-                displaySymbol: "%",
-                description: "This metric is based on the number of successful builds."
-            },
-	        {
-                metricName: "codeIssues",
-                displayName: "Code Violations",
-                scoreRanges: [{ rangeMin: 0, rangeMax: 0, score: 20 },
-                    { rangeMin: 1, rangeMax: 10, score: 10 },
-                    { rangeMin: 11, rangeMax: 100, score: 0 }],
-                displaySymbol: "",
-                description: "This metric is based on the number of source code issues discovered by Sonar."
-            },
-            {
-                metricName: "commitMessageMatch1",
-                displayName: "Fixed Builds",
-                scorePerCommit: 2,
-                commitMatchRegex: "FIX_BUILD",
-                displaySymbol: "",
-                description: "This metric is based on the number of commits that fixed broken builds. Note that the commit message must match the regular expression."
-            }
-        ];
 
         // method to toggle tabs
         function toggleView(index) {
             ctrl.widgetView = typeof ctrl.tabs[index] === 'undefined' ? ctrl.tabs[0].name : ctrl.tabs[index].name;
             if (ctrl.tabs[index].name == "Gamification") {
-                ctrl.populateScoreboardData();
-                $scope.chartData = ctrl.getChartData();
+                gamificationMetricData.getEnabledMetricData().then(ctrl.populateScoreboardData);
             }
         }
 
-        function populateScoreboardData() {
+        function defineChartProperties() {
+            $scope.chartOptions = {
+                plugins: [
+                    // Chartist.plugins.ctBarLabels({
+                    //     labelInterpolationFnc: function(text, value) {
+                    //         return value > 0 ? value : "";
+                    //     }
+                    // }),
```

Comment:
Also, we no longer plan to use this ctBarLabels, Can you please remove this commented code.